datapackagist/src/scripts/components/ui/upload.js

```diff
@@ -20,7 +20,8 @@ module.exports = backbone.BaseView.extend({
           }
           catch(E) { }
 
-          if(!_.isObject(descriptor)) {
+          // If descriptor is broken or If descriptor have field not from schema - reject it
+          if(!_.isObject(descriptor) || _.difference(_.keys(descriptor), _.keys(this.parent.layout.form.schema.properties)).length) {
```

Comment:
@gvidon this check for avoid upload json document with other structure. Required fields for upload not usually to check because it can be filled by user and validate later